File: cpp/Disk.cpp
#include <iostream>
#include <string>

using namespace std;

class Disk
{
private:
    string TypeDisk;
    int CapacityDisk;
    int PriceDisk;

public:
    Disk(string Type, int Capacity, int Price)
    {
        this->TypeDisk = Type;
        this->CapacityDisk = Capacity;
        this->PriceDisk = Price;
    }

    Disk() {}

    void setTypeDisk(string Type)
    {
        this->TypeDisk = Type;
    }

    string getTypeDisk()
    {
        return this->TypeDisk;
    }

    void setCapacityDisk(int Capacity)
    {
        this->CapacityDisk = Capacity;
    }

    int getCapacityDisk()
    {
        return this->CapacityDisk;
    }

    void setPriceDisk(int Price)
    {
        this->PriceDisk = Price;
    }

    int getPriceDisk()
    {
        return this->PriceDisk;
    }

    void print()
    {
        cout << "Disk :" << endl;
        cout << endl;
        cout << "Type -> " << this->getTypeDisk() << endl;
        cout << "Capacity (Gb) -> " << this->getCapacityDisk() << endl;
        cout << "Price (Rp) ->" << this->getPriceDisk() << endl;
    }

    ~Disk() {}
};

File: php/Disk.php
<?php

class Disk{
    private $NameDisk;
    private $CapacityDisk;
    private $PriceDisk;

    function __construct($Name, $Capacity, $Price)
    {
        $this->NameDisk = $Name;
        $this->CapacityDisk = $Capacity;
        $this->PriceDisk = $Price;
    }

    function setNameDisk($Name){
        $this->NameDisk = $Name;
    }

    function getNameDisk(){
        return $this->NameDisk;
    }

    function setCapacityDisk($Capacity){
        $this->CapacityDisk = $Capacity;
    }

    function getCapacityDisk(){
        return $this->CapacityDisk;
    }

    function setPriceDisk($Price){
        $this->PriceDisk = $Price;
    }

    function getPriceDisk(){
        return $this->PriceDisk;
    }

    function print(){
        echo "Disk :" . "<br/>";
        echo "Name :" . $this->getNameDisk(). "<br/>";
        echo "Capacity (Gb) :" . $this->getCapacityDisk(). "<br/>";
        echo "Price (Rp) :" . $this->getPriceDisk(). "<br/>";
    }
}
?>